pkg/fanal/types/const.go

```diff
@@ -30,6 +30,7 @@ const (
 	RustBinary = "rustbinary"
 	Conan      = "conan"
 	Cocoapods  = "cocoapods"
+	ElixirLock = "elixir-lock"
```

Comment:
Mix or Hex may be better. We can say we parse mix.lock and get Hex packages. Or, we parse mix.lock and get Mix dependencies.

Reply:
Done